bottle.py

```diff
@@ -2246,9 +2244,9 @@ def replace(self, key, value):
         self.dict[_hkey(key)] = [_hval(value)]
 
     def getall(self, key):
-        return self.dict.get(_hkey(key)) or []
+        return self.dict.get(_hkey(key), ) or []
 
-    def get(self, key, default=None, index=-1):
+    def get(self, key, default=None, index=-1, **kwargs):
```

Comment:
I mean I wanted to change this get() args matched with MultiDict.get() args.

Reply:
Ahh okay, I did not see that between all the tailing-comma-edits all over the place.

But it is not passed along to MultiDict.get() and not used, so why include it in the public API?